packages/sdk/src/index.ts

```diff
@@ -279,13 +279,21 @@ export default class SDK {
 
   /**
    * Creates a Request instance that can be sent through the RPCh network
-   * @param origin
    * @param provider
    * @param body
    * @returns Request
    */
   public createRequest(provider: string, body: string): Request {
     if (!this.isReady) throw Error("SDK not ready to create requests");
+    let entryNodeScore: number = this.reliabilityScore.getScore(
+      this.entryNode!.peerId
+    );
+    let counter: number = 0;
```

Comment:
what is the use of the counter?

Reply:
It's a limit of how many new entry nodes we would search for (the idea is to not create too much latency when creating the request)

Reply:
1. what happens when u reach 3?
2. this is not awaited so it wouldn't create latency

In general I don't think the implementation here is correct - I would rather update the discovery platform API to take in the `request/entry-node` an exclude list which would contain the previously bad entry node and give u a new entry node